### PR TITLE
updating caption

### PR DESCRIPTION
**Pull request by westonrp**
### Changed caption

- [ ] Pull request

### CODE REVIEW

**Comment by sandhyapabbu** [comment]
good

File: _slides/12-westonrp.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+This is my caption.
```
Comment (dmstoltz) [inline _slides/12-westonrp.md:6]: It is

**Review by sandhyapabbu — APPROVED** [review]

**Review by dmstoltz — APPROVED** [review]
Approved

File: _slides/12-westonrp.md (hunk 1 above)
Comment (westonrp) [inline _slides/12-westonrp.md:6]: looks good

**Review by westonrp — COMMENTED** [review]
looks good

**Pull request merged**